crates/swc_ecma_preset_env/src/corejs3/builtin.rs

```diff
@@ -173,506 +148,245 @@ static WEAK_SET_DEPENDENCIES: &[&str] = &concat2::<7>(
-  compositeSymbol: define("composite-symbol", ["esnext.composite-symbol"]),
-  escape: define("escape", ["es.escape"]),
-  fetch: define(null, PROMISE_DEPENDENCIES),
-  globalThis: define("global-this", ["es.global-this"]),
-  parseFloat: define("parse-float", ["es.parse-float"]),
-  parseInt: define("parse-int", ["es.parse-int"]),
-  queueMicrotask: define("queue-microtask", ["web.queue-microtask"]),
-  self: define("self", ["web.self"]),
-  setImmediate: define("set-immediate", ["web.immediate"]),
-  setInterval: define("set-interval", ["web.timers"]),
-  setTimeout: define("set-timeout", ["web.timers"]),
-  structuredClone: define("structured-clone", [
-    "web.structured-clone",
-    DOM_EXCEPTION_DEPENDENCIES,
-    "es.array.iterator",
-    "es.object.keys",
-    "es.object.to-string",
-    "es.map",
-    "es.set",
-  ]),
-  unescape: define("unescape", ["es.unescape"]),
+    DataView: define(
+        null,
+        [
+            "es.data-view",
+            "es.array-buffer.slice",
+            "es.object.to-string",
+        ]
+    ),
+    Date: define(null, ["es.date.to-string"]),
+    Float32Array: typed("es.typed-array.float32-array"),
+    Float64Array: typed("es.typed-array.float64-array"),
+    Int8Array: typed("es.typed-array.int8-array"),
+    Int16Array: typed("es.typed-array.int16-array"),
+    Int32Array: typed("es.typed-array.int32-array"),
+    Uint8Array: typed("es.typed-array.uint8-array"),
+    Uint8ClampedArray: typed("es.typed-array.uint8-clamped-array"),
+    Uint16Array: typed("es.typed-array.uint16-array"),
+    Uint32Array: typed("es.typed-array.uint32-array"),
+    Map: define("map/index", MAP_DEPENDENCIES),
+    Number: define(null, ["es.number.constructor"]),
+    Observable: define(
+        "observable/index",
+        [
+            "esnext.observable",
+            "esnext.symbol.observable",
+            "es.object.to-string",
+            COMMON_ITERATORS_WITH_TAG,
+        ]
+    ),
+    Promise: define("promise/index", PROMISE_DEPENDENCIES),
+    RegExp: define(
+        null,
+        [
+            "es.regexp.constructor",
+            "es.regexp.exec",
+            "es.regexp.to-string",
+        ]
+    ),
+    Set: define("set/index", SET_DEPENDENCIES),
+    Symbol: define("symbol/index", SYMBOL_DEPENDENCIES),
+    URL: define("url/index", ["web.url", URL_SEARCH_PARAMS_DEPENDENCIES]),
+    URLSearchParams: define("url-search-params/index", URL_SEARCH_PARAMS_DEPENDENCIES),
+    WeakMap: define("weak-map/index", WEAK_MAP_DEPENDENCIES),
+    WeakSet: define("weak-set/index", WEAK_SET_DEPENDENCIES),
+    clearImmediate: define("clear-immediate", ["web.immediate"]),
+    compositeKey: define("composite-key", ["esnext.composite-key"]),
+    compositeSymbol: define(
+        "composite-symbol",
+        ["esnext.composite-symbol", SYMBOL_DEPENDENCIES]
+    ),
+    fetch: define(null, PROMISE_DEPENDENCIES),
+    globalThis: define("global-this", ["es.global-this", "esnext.global-this"]),
```

Comment:
The source of these new codes are https://github.com/babel/babel/blob/e498bee10f0123bb208baa228ce6417542a2c3c4/packages/babel-preset-env/src/polyfills/corejs3/built-in-definitions.js but I'm not sure if this is correct.

Reply:
It's obsolete definitions. Actual definitions here https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts